src/userdev/java/net/minecraftforge/gradle/userdev/MinecraftUserRepo.java

```diff
@@ -1152,7 +1156,7 @@ private File findSource(String mapping, boolean generate) throws IOException {
 
                     if (name.endsWith(".java")) {
                         String mapped = map.rename(zin, addJavadocs && vanilla.contains(name.substring(0, name.length() - 5)));
```

Comment:
Need to pass in to McpNames, it defaults to UTF8 right now, but we need to make sure we use the correct encoding.
https://github.com/MinecraftForge/ForgeGradle/blob/c077542cef99ffadaf19b434eddf1e676f76c612/src/common/java/net/minecraftforge/gradle/common/util/McpNames.java#L106

Reply:
This call is easy to change but there is another use from `TaskApplyMappings` which currently has nothing to do with the user repo. What should be done about that call?

Reply:
You could add it as an argument to that task. However the current Patcher stuff is only using it for clean. Which should be UTF8. But it could probably use the full info from the parent patchers.